src/main/java/fr/esgi/al5_2/Tayarim/dto/proprietaire/AdministrateurDTO.java

```diff
@@ -0,0 +1,29 @@
+package fr.esgi.al5_2.Tayarim.dto.proprietaire;
+
+import fr.esgi.al5_2.Tayarim.dto.logement.LogementDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data // Generates getters, setters, toString, equals, and hashCode methods
+@AllArgsConstructor // Generates an all-args constructor
+@Builder
+public class AdministrateurDTO {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'AdministrateurDTO' must contain no more than '1' consecutive capital letters.